### PR TITLE
feat(fe/admin/users): add basic user table to admin

### PR DESCRIPTION
**Pull request by RickyLB**
https://github.com/ji-devs/ji-cloud/issues/3480

## Added
- `Users` tab
- table/list of basic user info (names)
## Fields
- `Username`
- `First Name`
- `Last Name`
## Note
- This is a temporary implementation for `Users` in the admin table
## Screenshot
admin user table

### CODE REVIEW

**Comment by RickyLB** [comment]
@MendyBerger Originally, I had gotten rid of the `curation` keyword in `admin-curation-table` element but using `admin-table-*` for both curation and users yielded conflicting elements. I added `user` and `curation` back into the element names.

File: frontend/apps/crates/entry/admin/src/users/actions.rs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    routes::{AdminRoute, AdminUsersRoute, Route},
+};
+
+use super::{editable_user::EditableUser, FetchMode, Users};
+
+impl Users {
+    pub fn load_data(self: &Rc<Self>) {
+        let state = Rc::clone(self);
+        state.loader.load(clone!(state => async move {
+            join!(
```
Comment (MendyBerger) [inline frontend/apps/crates/entry/admin/src/users/actions.rs:26]: No need to `join!` if you only have one future

File: frontend/apps/crates/entry/admin/src/users/user/state/mod.rs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+impl AdminUser {
+    pub fn new(users_state: Rc<Users>, user_id: UserId, user: Rc<EditableUser>) -> Rc<Self> {
+        Rc::new(Self {
+            user_id,
+            user,
+            loader: AsyncLoader::new(),
+            users_state,
+            player_open: Mutable::new(false),
```
Comment (MendyBerger) [inline frontend/apps/crates/entry/admin/src/users/user/state/mod.rs:23]: There's no player here, you can get rid of this

File: frontend/apps/crates/utils/src/routes.rs [hunk 3]
```diff
@@ -660,6 +673,13 @@ impl From<&Route> for String {
                         format!("/admin/curation/{}", jig_id.0)
                     }
                 },
+                AdminRoute::Users(users_route) => match users_route {
+                    AdminUsersRoute::Table => "/admin/users".to_string(),
+                    AdminUsersRoute::User(user_id) => {
+                        log::info!("routes.rs user: {:?}", user_id);
```
Comment (MendyBerger) [inline frontend/apps/crates/utils/src/routes.rs:679]: I'm guessing this is left over from debugging

**Review by MendyBerger — COMMENTED** [review]
@RickyLB this looks great! 

About the element name conflict, I meant that you should use the same element for both tables, not define another element with the same name.

**Pull request merged**